api/proto/teleport/usageevents/v1/usageevents.proto

```diff
@@ -194,6 +194,22 @@ message UIDiscoverDatabaseRDSEnrollEvent {
   int64 selected_resources_count = 4;
 }
 
+// UICallToActionClickEvent is emmited when a user clicks a Teleport Web UI's CTA.
+message UICallToActionClickEvent {
+  CTA cta = 2;
```

Comment:
It doesn't matter, but why start with message number 2?

Reply:
I think I copied a similar code from cloud and removed the field before without updating the cta number. Updated in https://github.com/gravitational/teleport/pull/25325/commits/3b7f9cdea7e5e01303f9a1be78ece95a9eb5bdae, thanks